src/framework/components/camera/system.js

```diff
@@ -107,6 +107,7 @@ class CameraComponentSystem extends ComponentSystem {
             clearDepthBuffer: c.clearDepthBuffer,
             clearStencilBuffer: c.clearStencilBuffer,
             cullFaces: c.cullFaces,
+            enabled: c.enabled,

```

Comment:
We probably need to do a full audit of all components to ensure they all correctly clone the `enabled` property. It looks like the `AnimComponent` doesn't clone it, for example.

Reply:
https://github.com/playcanvas/engine/issues/3413